doc/go-target.md

```diff
@@ -78,7 +78,7 @@ If you are already using the repo and import `github.com/antlr4-go/antlr/v4` the
 using the standard.
 
 ```shell
-go get -u get github.com/antlr4-go/antlr
+go get -u github.com/antlr4-go/antlr
```

Comment:
This change doesn't relate to the MR title.

Reply:
(Thanks. The focus was in the wrong window in Emacs, meant to be in a bash shell, and somehow saved it by mistake.)